README.md

```diff
@@ -37,6 +37,40 @@ _configs/config.json_
 }
 ```
 
+#### Sending config during run
+
+After starting the UI, config files can still by send in. To submit a new configuration, use the `ui_uri:port/config` REST API endpoint: send the configuration as `json` payload in a `POST` request. Please note that currently the whole configuration must be submitted, the old one will be replaced.
+
+Submission example using python:
+```python
+import requests
+config = { ... }
```

Comment:
Please add a dummy config, so people know what is the root tag of the json and what is expected specifically

Reply:
Added.